proto/server/configEvtHandler.go

```diff
@@ -210,6 +215,34 @@ func handleNetworkSlicePost(configMsg *configmodels.ConfigMessage, subsUpdateCha
 	if errPost != nil {
 		logger.DbLog.Warnln(errPost)
 	}
+	if factory.WebUIConfig.Configuration.SendPebbleNotifications {
+		err := sendPebbleNotification("canonical.com/webconsole/networkslice/create")
```

Comment:
Along with Notification configuration, take the target base url configuration as well. So hardcoding specific to company name can be avoided.

Reply:
This string is here just for namespacing the notifications. I really don't think it's worth adding another config param for it. This param will also need some validation, handling for the case when it's missing. 
I just committed a proposal to use `aetherproject.org` as the base URL to avoid "promoting" external companies ;) Would that work?

Reply:
Thanks